src/commands.js

```diff
@@ -750,19 +750,10 @@ module.exports = {
       },
     },
   },
-  tokenLookupAccessor: {
+  tokenLookupSelf: {
     method: 'POST',
```

Comment:
Per the API docs, shouldn't this be a `GET`?

https://www.vaultproject.io/api-docs/auth/token#lookup-a-token-self